packages/react-vapor/src/components/radio/Radio.tsx

```diff
@@ -2,15 +2,39 @@ import classNames from 'classnames';
 import * as React from 'react';
 import {IInputProps, Input} from '../input/Input';
 
-export class Radio extends Input {
-    static defaultProps: Partial<IInputProps> = {
+export interface RadioOwnProps {
+    outerContainerClass?: string;
+    outerElementInContainer?: React.ReactNode;
+}
+
+export interface RadioProps extends RadioOwnProps, IInputProps {}
+
+export class Radio extends React.PureComponent<RadioProps> {

```

Comment:
I know you probably took example on other components for your `pureComponent` here but I think that would be cleaner and faster with a function component instead than a class component. And since it's stateless, you will not have any difficulty doing so.

You can do something like that 
```
export const Radio: React.FunctionComponent<RadioProps> = (props) => {
[... then here you will have consts instead of class static and private elements and instead of a render() method you are going to return the jsx directly]
}
```